clippy_lints/src/missing_doc.rs

```diff
@@ -52,6 +53,22 @@ impl MissingDoc {
         *self.doc_hidden_stack.last().expect("empty doc_hidden_stack")
     }
 
+    #[allow(clippy::needless_bool)]
+    fn has_include(meta: Option<MetaItem>) -> bool {
+        if_chain! {
+            if let Some(meta) = meta;
+            if let MetaItemKind::List(list) = meta.node;
+            if let Some(meta) = list.get(0);
+            if let Some(name) = meta.name();
+            if name == "include";
```

Comment:
I believe this can be rewritten as:

```rust
then {
    name == "include"
}
```

Reply:
Done.
https://github.com/rust-lang/rust-clippy/blob/e1096e3310e414e432884b7b38de8bef26779b7b/clippy_lints/src/missing_doc.rs#L63